runtime/compiler/control/JitDump.cpp

```diff
@@ -483,9 +483,86 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
 
    try
       {
-      // if our compinfo is null, we are an application thread
+      // Process user defined options first. The user can specify any method signature to be recompiled at an arbitrary
+      // dump event. This is typically used for debugging purposes so we process this option first if it exists.
+      if (agent->dumpOptions != NULL)
+         {
+         char *indexOfDot = strchr(agent->dumpOptions, '.');
+         char *indexOfParen = strchr(agent->dumpOptions, '(');
+
+         if ((NULL != indexOfDot) && (NULL != indexOfParen))
+            {
+            TR::VMAccessCriticalSection findUserMethod(TR_J9VMBase::get(jitConfig, crashedThread));
+
+            char *className = agent->dumpOptions;
+            U_32 classNameLength = indexOfDot - agent->dumpOptions;
+            const char *methodName = indexOfDot + 1;
+            U_32 methodNameLength = indexOfParen - indexOfDot - 1;
+            const char *methodSig = indexOfParen;
+            U_32 methodSigLength = strlen(indexOfParen);
+
+            J9MethodFromSignatureWalkState state;
+            J9Method *userMethod = allMethodFromSignatureStartDo(&state, context->javaVM, 0, className, classNameLength, methodName, methodNameLength, methodSig, methodSigLength);
+            while (userMethod != NULL)
```

Comment:
Please consistently put the constant on the left (`NULL != userMethod`) like on line 493.
See also lines 488, 509, 516, etc.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/70f787d25b95b995346cd4b3b411e35100b48f98..e6d4272c33428d33d17da82f1cddbbf549d82084).